qa/standalone/ceph-helpers.sh

```diff
@@ -476,10 +476,15 @@ function test_run_mon() {
 
 function create_rbd_pool() {
     ceph osd pool delete rbd rbd --yes-i-really-really-mean-it || return 1
-    ceph osd pool create rbd $PG_NUM || return 1
+    create_pool rbd $PG_NUM || return 1
     rbd pool init rbd
 }
 
+function create_pool() {
+    ceph osd pool create "$@"
+    sleep 1
```

Comment:
This seems fragile.  It seems like we need a mechanism to synchronously wait for the mgr to get the latest osdmap.  Or query which epoch is has to we can poll until it does.  This will work for these tests as long as the test machine isn't under load since there is no failure injection with these tests, but... there will be some sporatic failures.